docs/source/model_design/vivarium_model_components/fertility/index.rst

```diff
@@ -0,0 +1,134 @@
+----------------------
+
+Pregnancy and births are not explicit outcomes in the GBD study. However, there are location- and year-specfic GBD covariates related to fertility and births which we can use to calculate the number
+of new simulants to add per timestep.
+
+.. list-table:: Covariates
+  :widths: 15 15 15
+  :header-rows: 1
+
+  * - Covariate name
+    - Covariate ID
+    - Note
+  * - Live births by sex
+    - 1106
+    - 
+
+
+Vivarium Modeling Strategy
+----------------------------
+
+In a simulation where we do not model births by identifying which simulants give birth, we must instead use live births data
+to determine how many simulants to add in each time step.
+We will scale down the live births of the total GBD population by the same "model scale"  we use in our simulation.
+
+For example, imagine that country A has a GBD population of 1000, and we are modeling this total population, but only need to initialize children age 5 and under 
+(e.g. because they are the only ones susceptible to the disease we are modeling).
+Imagine there are 100 such children living in A, and we plan to initialize 20 children in our model. 
+
+In this example, our initialized population is :math:`\frac{1}{5}` the size of the age-restricted GBD population it corresponds to. 
+This means that the simulated population implied by our model scenario (which includes people over the age of 5 even though they are not initialized) is also :math:`\frac{1}{5}` 
+the size of the total unrestricted GBD population. 
+This is our "model scale". We want :math:`\frac{1}{5}` as many children to be born in our simulation as are born in the total GBD population.
+
+.. note::
+  Even though our number of simulants is much less than :math:`\frac{1}{5}` of the **total** GBD population of 1000 (since we only need to model children), we still want
+  :math:`\frac{1}{5}` as many births to occur in our simulation as in the total GBD population. 
+  
+  The reason is that all ages of people still exist in the scenario we are modeling, despite the fact that we are only 
+  explicitly initializing simulants corresponding to children under 5 -- otherwise, there would be no parents to give birth!
+  Therefore, our model scale must reflect the total GBD population even if we are only initializing a subset of that population in our model. 
+  
+  In other words, the initialized population is different from the population that exists within the simulation scenario. 
+  The ratio of the (age-restricted) initialized population to the age-restricted GBD population gives us our model scale.
+  The ratio of the simulation scenario population to the total GBD population is also equal to the model scale 
+  (in this example the ratio would be 200 to 1000). 
+
+To represent model scale in general we can define the below equation:
+
+:math:`\text{model scale} = \frac{\text{Initialized population size}}{\int_{a=\text{age start}}^{\text{age end}} \text{GBD total population}(a) \, da}`
```

Comment:
I think the units of this integral don't make sense: population $\cdot$ age = person-time, not population. I.e., the integrand should be population per year or something like that instead of just population, since you're multiplying by $da$. I think it would probably be better to write it as a sum over age groups, which is how we'd actually compute it.